applications/GeoMechanicsApplication/tests/cpp_tests/custom_geometries/test_interface_geometry.cpp

```diff
@@ -353,4 +354,60 @@ KRATOS_TEST_CASE_IN_SUITE(InterfaceGeometry_Throws_WhenCallingVolume, KratosGeoM
                                       "one. Please check the definition of derived class. ")
 }
 
+KRATOS_TEST_CASE_IN_SUITE(InterfaceGeometry_Throws_WhenCallingFunctionsRelatedToIntegrationPoints,
+                          KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    auto                          geometry = CreateThreePlusThreeNodedLineInterfaceGeometry();
+    Geometry<Node>::JacobiansType dummy_jacobian;
+    Geometry<Node>::ShapeFunctionsGradientsType dummy_shape_functions_gradients;
+    Vector                                      dummy_vector;
+    Matrix                                      dummy_matrix;
+    IndexType                                   dummy_index = 0;
+    GeometryData::IntegrationMethod dummy_integration_method = GeometryData::IntegrationMethod::GI_GAUSS_1;
+    Geometry<Node>::IntegrationPointsArrayType dummy_integration_points;
```

Comment:
Confusing is the <Node> here for the template parameter as we are creating integration points.

Reply:
If you mean the template parameter of Geometry (so Node): the input parameter for some of these functions refer to this IntegrationPointsArrayType, which is an alias in Geometry, meaning normally if we want to get that type, we'd type `Geometry::IntegrationPointsArrayType`. However, since Geometry is a templated class, we always need to specify the template argument when retrieving something from it, since it could in principle be, that the IntegrationPointsArrayType depends on what kind of points we use for the Geometry (in this case Node). For IntegrationPoints that's not the case, but that's the reason we need to specify the template parameter when calling on Geometry

Reply:
Just meant that it reads unexpected that the template parameter is node, whilst we are creating integration points here. Nothing blocking.